app/jsx/components/TabContent1Comp.jsx

```diff
@@ -11,69 +11,108 @@ class TabContent1Comp extends React.Component {
     let p = this.props
     return (
       <div className="c-tabcontent">
-        <ItemActionsComp />
+        <ItemActionsComp status={p.status} content_type={p.content_type} id={p.id} />
         <h1 className="c-tabcontent__heading">{p.title}</h1>
-        <AuthorListComp pubdate={p.pub_date} authors={p.authors} changeTab={this.props.changeTab} />
-        <PublishedLocationComp loc={p.attrs["pub_web_loc"]} />
-        {p.attrs["abstract"] &&
-          <details className="c-togglecontent" open>
-            <summary><h2>Abstract</h2></summary>
-            <p>{p.attrs["abstract"]}</p>
-            {/* ToDo: Determine how this content is coming in */}
-            <p className="c-well">***OPEN ACCESS POLICY*** Libero dolores rerum nesciunt deserunt incidunt, aspernatur similique fugit beatae quis impedit corrupti, voluptate, unde facilis. Voluptatibus labore sunt maxime, accusantium animi mollitia ducimus.</p>
-          </details>
-        }
-        <Content {...p} />
+        <AuthorListComp pubdate={p.pub_date} authors={p.authors} changeTab={p.changeTab} />
+        <PublishedLocationComp pub_web_loc={p.attrs.pub_web_loc} rights={p.rights} />
+        {this.props.attrs.abstract && (this.props.status != "withdrawn") &&
+          <Abstract status={p.status} abstract={p.attrs.abstract} /> }
+        <MainText {...p} />
       </div>
     )
   }
 }
 
-class Content extends React.Component {
+class Abstract extends React.Component {
   render() {
-    switch(this.props.status) {
+    return (
+      <details className="c-togglecontent" open>
+        <summary><h2>Abstract</h2></summary>
+        <p>{this.props.abstract}</p>
+        {/* ToDo: Add Link */}
+        <p className="c-well">Many UC-authored scholarly publications are freely available on this site because of the UC Academic Senate&apos;s Open Access Policy. *** LINK *** Let us know how this access is important for you.</p>
+      </details>
+    )
+  }
+}
+
+class MainText extends React.Component {
+  render() {
+    let p = this.props
+    if (!p.content_type) return (<NoContent pub_web_loc={p.attrs.pub_web_loc} />)
+    switch(p.status) {
       case "published":
-        return this.renderContent(this.props)
-        break;
+        return (p.content_type == "application/pdf" ? this.renderPdf(p) :
+              p.content_type == "text/html" ? this.renderHtml(p) : null)
+        break
```

Comment:
There's no need for `break` after `return`, because `return` immediately exits the function and the `break` never gets executed.